conf/up4.bess

```diff
@@ -227,6 +228,32 @@ if parser.enable_slice_metering:
     sliceMeter.set_default_gate(gate=m_fail)
     _in = sliceMeter
 
+# TODO: decide whether UE policing module is optional
```

Comment:
Optionality of UE metering is up to the control plane, so I think the module should always be enabled

Reply:
Sounds good. The current wiring allows all no-matching packets to pass through, implementing optionality.